client/src/pages/Footer/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import { classnames } from "../../Utilities";
+import { Routes, Route, BrowserRouter, Link } from "react-router-dom";
+
+export default function Footer() {
```

Comment:
Good job modularizing this component. Please put it inside the `components` directory rather than the `pages` directory because that's what it is

Reply:
Yes, I have moved it to the component folder and updated all the imports!